### PR TITLE
Default enrollment API auto_enroll to True

### PR DESCRIPTION
**Pull request by stvstnfrd**
since this is what we designed the API for..

### CODE REVIEW

**Comment by stvstnfrd** [comment]
BTW: SCPD is the only group using this and they always want this value set to True,
so no regression here.

File: openedx/stanford/common/djangoapps/enrollment/views.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def delete(self, request, course_id):
                 },
             )
         email_students = request.data.get('email_students', False) in ['true', 'True', True]
-        auto_enroll = request.data.get('auto_enroll', False) in ['true', 'True', True]
+        auto_enroll = request.data.get('auto_enroll', True) in ['true', 'True', True]
         email_params = {}
         language = None
         if email_students:
```
Comment (dcadams) [inline openedx/stanford/common/djangoapps/enrollment/views.py]: Looking more widely at this code, it's a bit inefficient to query the User table twice just to get a single user. You can do it with a single db hit like so:

try:
    user = User.objects.get(email=email)
except User.DoesNotExist:
    pass
else:
    language = get_user_email_language(user)

File: openedx/stanford/common/djangoapps/enrollment/views.py [hunk 2]
```diff
@@ -160,7 +160,7 @@ def delete(self, request, course_id):
                 },
             )
         email_students = request.data.get('email_students', False) in ['true', 'True', True]
-        auto_enroll = request.data.get('auto_enroll', False) in ['true', 'True', True]
+        auto_enroll = request.data.get('auto_enroll', True) in ['true', 'True', True]
```
Comment (dcadams) [inline openedx/stanford/common/djangoapps/enrollment/views.py]: This looks ok.

**Comment by dcadams** [comment]
Somewhat tangential, I wrote a unit test for the roster api that, as far as I can fathom, has gone missing:

https://github.com/Stanford-Online/edx-platform/pull/509

The commit is in the history: 94fd7d8554f63d132b6587c55a753c88bb31d973
but I can't find the code.

File: openedx/stanford/common/djangoapps/enrollment/views.py (hunk 1 above)
Comment (dcadams) [reply openedx/stanford/common/djangoapps/enrollment/views.py]: @stvstnfrd You can close this PR in favour of my one.

**Comment by stvstnfrd** [comment]
Closing in favor of #841

**Pull request closed without merging**